src/library/cratefeature.h

```diff
@@ -49,8 +51,12 @@ class CrateFeature : public LibraryFeature {
     void activateChild(const QModelIndex& index);
     void onRightClick(const QPoint& globalPos);
     void onRightClickChild(const QPoint& globalPos, QModelIndex index);
+    void onLazyChildExpandation(const QModelIndex& index);
+    void itemCollapsed(const QModelIndex& index);
 
-    void slotCreateCrate();
+    void slotCreateCrate(int crateType);
+    inline void slotCreateNormalCrate() { slotCreateCrate(0); } ;
```

Comment:
NormalCrate is crate. Not folder not anything else.
This function is just shortcut. It is possible to everywhere instead of slotCreateNormalCrate() slotCreateCrate(0)